src/components/CountryInfo.js

```diff
@@ -129,7 +129,7 @@ export class CountryInfo extends Component {
           <h2>Stats</h2>
```

Comment:
There's an unused check now: https://github.com/Code-for-All/lockdown/blob/7f8e86f7e6f383c1ad9d75dea8449d692db4b937/src/components/CountryInfo.js#L96

Have you tested this when the user doesnt have internet connection, too?

Reply:
I tested and it didn't display the offline message

Reply:
it should display the offline message instead when a user is offline

Reply:
yup, will fix that